src/style.css

```diff
@@ -1 +1,87 @@
-/* add styles here */
+@font-face {
+  font-weight: 400;
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.search {
+  display: flex;
+  flex-direction: column;
+  margin: 71px 81px 482px 48px;
+}
+
+.search__panel {
+  display: flex;
+  align-items: center;
+  background: #FFF;
+  border-radius: 4px;
+  border: 1px solid #E1E7ED;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.10);
+  color: #3D4E61;
+}
+
+.search__panel--big {
+  height: 70px;
+}
+
+.search__panel--small {
+  height: 42px;
+  margin-top: 20px;
+}
+
+.search__panel:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.20);
+}
+
+.search__panel:focus {
+  border-radius: 3px;
+  border: 1px solid #E1E7ED;
+  background: linear-gradient(180deg, #FFF 0%, #F6F6F7 100%);
+}
+
+.search__input {
```

Comment:
pay attention to text color

also, note that you should also add styles to placeholder (use `:placeholder`)